cassandra-migration/src/main/java/org/cognitor/cassandra/migration/util/ChecksumUtil.java

```diff
@@ -0,0 +1,12 @@
+package org.cognitor.cassandra.migration.util;
+
+import java.util.zip.CRC32;
+
+public class ChecksumUtil {
+
+	public static int calculateCRC32(String script) {
+        final CRC32 crc32 = new CRC32();

```

Comment:
I was not aware that CRC32 can be used for this, so thanks that I could learn something :) But I don't think that it is a good idea to cast the result explicitly to int. The reason for that is that you know internals of the CRC32 implementation that could in theory change at any time. Usually you should never rely on the internal implementation of a class.